libraries/podofo/podofo.SlackBuild

```diff
@@ -80,36 +81,28 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \+
 
-sed -i "s/BINARY_DIR}\/objects\"/BINARY_DIR}\/objects\" || true/" test/TokenizerTest/CMakeLists.txt
-
-# Fix mandir directory.
-sed -i 's|share/man|man|' CMakeLists.txt
-# Ref: https://sourceforge.net/p/podofo/tickets/28/
-patch -p1 < $CWD/XRefStrParObj-deprecated.patch
```

Comment:
Delete the actual patch file too?

Reply:
Thanks Andrew - I now have deleted the patch file.